INSTALL.md

```diff
@@ -62,7 +62,7 @@ $ php app/console ngadminui:install
 This will install all required configuration in two places:
 
 * `app/config/ngadminui.yml`
-* `ezpublish_legacy/siteaccess/NEW_SITEACCESS_NAME`
+* `ezpublish_legacy/settings/siteaccess/NEW_SITEACCESS_NAME`
```

Comment:
Isn't this already merged? :)

You can rebase the pull request to get rid of this diff.